tests/Endpoints/SchoolTest.php

```diff
@@ -44,10 +44,10 @@ protected function getFixtures(): array
     public function putsToTest(): array
     {
         return [
-            'iliosAdministratorEmail' => ['iliosAdministratorEmail', $this->getFaker()->email()],
-            'title' => ['title', $this->getFaker()->text(60)],
-            'templatePrefix' => ['templatePrefix', $this->getFaker()->text(8)],
-            'changeAlertRecipients' => ['changeAlertRecipients', $this->getFaker()->email()],
+            'iliosAdministratorEmail' => ['iliosAdministratorEmail', 'lorem.ipsum@dev.null'],
```

Comment:
I think at anything less than a million bucks owning the TLD .null and creating email addresses like `lore.ipsum@dev.null` that only get spam and then use AI to waste the spammers time would be worth it and probably make us rich!